src/main/java/cz/student/jankocabek/TaskManager.java

```diff
@@ -1,9 +1,198 @@
+        final var sizeTaskData = 3;
+        final int taskNum = tasks.length + 1;
+        final String[] task = new String[sizeTaskData];
+
+        System.out.printf("\n%sAdding new task Number.%d:%s\n", GREEN_BOLD_BRIGHT, taskNum, RESET);
+        System.out.printf("%sWrite the description of the task:%s ", BLUE_BOLD_BRIGHT, RESET);
+        task[0] = scanner.nextLine();
+
+        System.out.printf("%sWrite the date when task end -in the format %sYYYY-MM-DD%s:%s ", BLUE_BOLD_BRIGHT, YELLOW_BOLD_BRIGHT, BLUE_BOLD_BRIGHT, RESET);
+        task[1] = " %s".formatted(loadAndCheckDate());//because space after comma
+
+        System.out.printf("%sWrite \"%strue%s\" or \"%sfalse%s\" if task is important or not:%s ", BLUE_BOLD_BRIGHT, YELLOW_BRIGHT, BLUE_BOLD_BRIGHT, YELLOW_BOLD_BRIGHT, BLUE_BOLD_BRIGHT, RESET);
+        while (!scanner.hasNext("true") && !scanner.hasNext("false")) {
+            scanner.nextLine();//for cleaning buffer before new try
+            System.out.printf("\n%sInvalid option - put only %strue%s or %sfalse%s:%s ", RED_BOLD_BRIGHT, YELLOW_BOLD_BRIGHT, RED_BOLD_BRIGHT, YELLOW_BOLD_BRIGHT, RED_BOLD_BRIGHT, RESET);
+        }
+        task[2] = " %s".formatted(scanner.next());//next only because if somebody write something after true/false then would catch also
+        scanner.nextLine();//cleaning buffer
+        tasks = ArrayUtils.add(tasks, task);
+        System.out.printf("%sTask  Number %d added%s\n", GREEN_BOLD_BRIGHT, taskNum, RESET);
+    }
+
+    //my method for controlling if string is in right format and the date is at least today
+    private static String loadAndCheckDate() {
+
+
+        while (true) try {
+            final var today = LocalDate.now();
+            final String input = scanner.nextLine();
+            final var date = LocalDate.parse(input);
+            //date must be at least today if so method end and return input
+            if (!date.isBefore(today)) return input;
+            System.out.printf("%sThis date already was!!%s\n", RED_BOLD_BRIGHT, RESET);
+            System.out.printf("\n%sWrite date which is at least %sToday:%s ",
+                    BLUE_BOLD_BRIGHT, RED_BOLD_BRIGHT, RESET);
+        } catch (DateTimeParseException e) {
+            System.out.printf("%sInvalid date format!! %s\n", RED_BOLD_BRIGHT, RESET);
+            System.out.printf("\n%sWrite date in the format %sYYY-MM-DD%s:%s ",
+                    BLUE_BOLD_BRIGHT, YELLOW_BOLD_BRIGHT, BLUE_BOLD_BRIGHT, RESET);
+        }
+    }
+
+    private static void removeTask() {
+        String input;
+
+        System.out.printf("\n%sWrite the number of the task you want to remove from the list%s\n", BLUE_BOLD_BRIGHT, RESET);
+        System.out.printf("%sif you don't know the number of the task write \"%s?%s\" or \"%sexit%s\" to return to the main menu:%s", BLUE_BOLD_BRIGHT, YELLOW_BOLD_BRIGHT, BLUE_BOLD_BRIGHT, YELLOW_BRIGHT, BLUE_BOLD_BRIGHT, RESET);
+        do {
+            input = scanner.nextLine().toLowerCase();
```

Comment:
CLEAN-CODE: promennou input inicializuj az v do-while cyklu, navrhu neni potreba.

```suggestion
final var input = = scanner.nextLine().toLowerCase();
```